node/src/GraphBuilder.cpp

```diff
@@ -61,6 +62,24 @@ Napi::FunctionReference node::GraphBuilder::constructor;
     operand->SetImpl(mImpl.op(input));                                                      \
     return object;
 
+#define BUILD_ACTIVATION_UNARY(op)                                                              \
```

Comment:
Change this macro to `BUILD_UNARY_OPERATOR` and only keep the code to build operator.